google/cloud/spanner/transaction.h

```diff
@@ -20,23 +20,20 @@
 #include "google/cloud/spanner/version.h"
 #include <google/spanner/v1/transaction.pb.h>
 #include <chrono>
-#include <functional>
 #include <memory>
 
 namespace google {
 namespace cloud {
 namespace spanner {
 inline namespace SPANNER_CLIENT_NS {
-
```

Comment:
How come your environment likes to delete these lines, but mine doesn't and the clang-tidy build doesn't complain about it?

Reply:
Short answer is I do not know.  The longer answer: clang-format does not seem to care if you have 0 or 1 lines there. It cares if you have 2 or more, I suspect my setup at home just wants 0.